### PR TITLE
feat: migrate to `ember-showdown-shiki`

### PR DESCRIPTION
**Pull request by IgnaceMaes**
The last one to align all the Ember apps?

Migrates to: https://github.com/IgnaceMaes/ember-showdown-shiki

This is a breaking change due to config required and node version bump.

Preview: https://deploy-preview-18--rfc-process-mdbook-template.netlify.app/id/0001-first-rfc/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,9 +30,10 @@
     "@fortawesome/ember-fontawesome": "https://git@github.com/mansona/ember-fontawesome.git#update-get-config",
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "broccoli-funnel": "^2.0.2",
+    "ember-auto-import": "^2.0.0",
     "ember-cli-babel": "^7.26.10",
     "ember-cli-htmlbars": "^5.7.2",
-    "ember-showdown-prism": "^3.0.0"
+    "ember-showdown-shiki": "^1.2.1"
```
Comment (mansona) [inline package.json:36]: isn't this creating an implicit peer on showdown? do you think we should make it explicit here too?

**Review by mansona — APPROVED** [review]
All in all a great improvement 🎉 one question about peers but not a blocker

File: package.json (hunk 1 above)
Comment (IgnaceMaes) [reply package.json:36]: It does! ([ref](https://github.com/IgnaceMaes/ember-showdown-shiki/blob/main/ember-showdown-shiki/package.json#L112-L114))

But `ember-cli-showdown` isn't. I'm guessing that somehow also was an implicit peer of `ember-showdown-prism`? As that one needed to be added explicitly to make it work.

Comment (mansona) [reply package.json:36]: yea maybe 🤷 its tough to know what the right thing to do with peers like this is 🙈 do you have any preference? should we leave it as it is?

Comment (IgnaceMaes) [reply package.json:36]: Not too sure on this one, to be honest.

But if not required, I would just add it explicitly in the consuming `rfc-app`. I think that's how we do it for the other apps too?

Comment (mansona) [reply package.json:36]: yes but I'm more asking if we should explicitly state it as a peer here. You're right that consuming apps would need to define it explicitly 👍 but that would be true regardless if we state it as a peer at this level

**Comment by mansona** [comment]
so we will need to rebase this now that I merged https://github.com/empress/rfc-process-mdbook-template/pull/19 to fix CI 👍

File: package.json (hunk 1 above)
Comment (IgnaceMaes) [reply package.json:36]: Ah I see what you're saying. Yes let's do that.

Added.

**Comment by IgnaceMaes** [comment]
Great!

Rebased so this should be good now.

**Review by mansona — APPROVED** [review]
:shipit:

**Pull request merged**